src/corecel/sys/ActionInterface.hh

```diff
@@ -210,15 +211,27 @@ class StepActionInterface : public ActionTypeTraits<P, S>,
       using ConcreteAction::ConcreteAction;
   };
  * \endcode
+ *
+ * The \c noexcept declarations improve code generation for the
+ * common use case of multiple inheritance.
+ *
+ * \note Use this class when multiple instances of the class may coexist in the
+ * same stepping loop.
```

Comment:
Do we have any such actions in our loop?

Reply:
I thought one of the "calo" or step classes did but apparently not. Since the "step collector" class has different use cases (hit manager and root step writer collect different data), that will eventually create different actions I think.